Later pytest versions have dependencies on packages, which doesn't longer work with Python 2.7
--- cryptography-2.6.1/setup.py	2019-02-28 02:27:53.000000000 +0000
+++ setup.py	2020-03-13 12:58:03.691136494 +0000
@@ -54,7 +54,7 @@
         )
 
 test_requirements = [
-    "pytest>=3.6.0,!=3.9.0,!=3.9.1,!=3.9.2",
+    "pytest==3.6.4",
     "pretend",
     "iso8601",
     "pytz",
